browse/scripts/main.py

```diff
@@ -39,18 +39,9 @@
 # topics
 tp = TopicsParser()
 topics = tp.gen_topics("../../database/db.csv")
+topics_file = open("../topics/topics-list.md", "w", encoding="UTF-8")
```

Comment:
in the `develop` `HEAD` this file is renamed to `00-topics-list.md`. Please keep the name unchanged. Once you pull `develop` from the upstream, you will see it.

Reply:
Yea I'm going to make a new PR with the change that were on develop.